features/liquidity/components/LiquidityBreakdown.tsx

```diff
@@ -103,10 +125,7 @@ export const LiquidityBreakdown = ({
   const compactTokenAAmount = formatCompactNumber(tokenAAmount, 'tokenAmount')
   const compactTokenBAmount = formatCompactNumber(tokenBAmount, 'tokenAmount')
 
-  const formattedTotalLiquidity = dollarValueFormatterWithDecimals(
-    totalLiquidity?.dollarValue,
-    { includeCommaSeparation: true }
-  )
+  const formattedTotalLiquidity = formatCompactNumber(totalLiquidity?.dollarValue) 
```

Comment:
do we no longer want the commaSeparation?

Reply:
Hey, we are using `M` and `B` for millions and billions respectively. It's more compact!